notebooks/integrations/integration-sklearn.ipynb

```diff
@@ -325,19 +327,39 @@
    "metadata": {},
    "outputs": [
     {
-     "name": "stderr",
+     "name": "stdout",
      "output_type": "stream",
      "text": [
-      " * Running on http://127.0.0.1:8050/ (Press CTRL+C to quit)\n",
-      "127.0.0.1 - - [16/Apr/2021 16:04:20] \"\u001b[37mGET /_alive_6db69162-e624-4856-9277-470869342b0b HTTP/1.1\u001b[0m\" 200 -\n"
+      "Dash is running on http://127.0.0.1:8050/\n",
+      "\n",
+      " * Serving Flask app 'rubicon_ml.viz.base' (lazy loading)\n",
+      " * Environment: production\n",
+      "\u001b[31m   WARNING: This is a development server. Do not use it in a production deployment.\u001b[0m\n",
+      "\u001b[2m   Use a production WSGI server instead.\u001b[0m\n",
+      " * Debug mode: off\n"
      ]
     },
     {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "Dash app running on http://127.0.0.1:8050/\n"
-     ]
+     "data": {
+      "text/html": [
+       "\n",
+       "        <iframe\n",
+       "            width=\"100%\"\n",
+       "            height=\"600px\"\n",
+       "            src=\"http://localhost:8050/\"\n",
+       "            frameborder=\"0\"\n",
+       "            allowfullscreen\n",
+       "            \n",
+       "        ></iframe>\n",
+       "        "
+      ],
+      "text/plain": [
+       "<IPython.lib.display.IFrame at 0x1a4d01ba0>"
+      ]
+     },
```

Comment:
can you also remove this `data` block like you did with the warning? this IFrame will render weird in the docs